ci-operator/step-registry/configure-registry-storage/set-nutanix-object/configure-registry-storage-set-nutanix-object-commands.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This script will set image registry to use nutanix object storage. 
+# It need deploy against nutanix platform.
+# Since nutanix object service still not support terraform, so we will use fixed bucket accordint to OCP version. And will clean up data on nutanix web console periodically.
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/xiuwang-nutanix-object-cred
+
+AWS_ACCESS_KEY_ID=$(cat "${AWS_SHARED_CREDENTIALS_FILE}" | grep aws_access_key_id | tr -d ' ' | cut -d '=' -f 2) && export AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY=$(cat "${AWS_SHARED_CREDENTIALS_FILE}" | grep aws_secret_access_key | tr -d ' ' | cut -d '=' -f 2) && export AWS_SECRET_ACCESS_KEY
+
+if [[ ${AWS_ACCESS_KEY_ID} == "" ]] || [[ ${AWS_SECRET_ACCESS_KEY} == "" ]]; then
+  echo "Did not find AWS credential, exit now"
+  exit 1
+fi
+
+OCP_MINOR_VERSION=$(oc version | grep "Server Version" | cut -d '.' -f2)
+NUTANIX_ENDPOINT="https://os.lts-cluster.internal.nutanix-dev.devcluster.openshift.com"
```

Comment:
This is hard code, suggest to save the info to vault, @sgaoshang can you help work with @xiuwang to save the endpoint to value, then get the value from vault?

Reply:
Sure, @xiuwang do you have permission to update it in vault? Anything else needed, pls let me know.

Reply:
@sgaoshang Seem I have no permission of vault, could you help me update it?  Thanks